dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 git+https://github.com/dbt-labs/dbt-adapters.git@main#subdirectory=dbt-adapters
 git+https://github.com/dbt-labs/dbt-adapters.git@main#subdirectory=dbt-tests-adapter
-git+https://github.com/dbt-labs/dbt-common.git@main
+git+https://github.com/dbt-labs/dbt-common.git@qmalcolm--292-add-adapter-type-to-invocation-context
```

Comment:
This should be reverted before merging. ADDITIONALLY, we need to:
1. Merge this [dbt-common PR](https://github.com/dbt-labs/dbt-common/pull/297)
2. Release dbt-common (likely a minor bump)
3. Bump the minimum allowed dbt-common in `setup.py`